remoteworkitem/trackerquery_repository_test.go

```diff
@@ -106,6 +107,7 @@ func doWithTrackerRepositories(t *testing.T, todo func(trackerRepo application.T
 		trackerRepo := NewTrackerRepository(db)
 		queryRepo := NewTrackerQueryRepository(db)
 		todo(trackerRepo, queryRepo)
+		gormsupport.DeleteCreatedEntities(db)
```

Comment:
This is not how this works. Is should be called as "defer gormsupport.DeleteCreatedEntities(db)()" in the start of a method.